tests/network/general/test_dhcp6_configurations_on_ipv4_single_stack_cluster.py

```diff
@@ -2,28 +2,28 @@
 
 import pytest
 
-from utilities.constants import OS_FLAVOR_CIRROS, Images
+from utilities.constants import OS_FLAVOR_FEDORA, Images
 from utilities.infra import get_node_selector_dict
-from utilities.virt import CIRROS_IMAGE, VirtualMachineForTests
+from utilities.virt import FEDORA_IMAGE, VirtualMachineForTests
 
 DHCPV6_PORT = 547
-VM_CIRROS = "vm-cirros"
+VM_FEDORA = "vm-fedora"
 LOGGER = logging.getLogger(__name__)
 
 
 @pytest.fixture(scope="module")
-def vm_cirros(
+def vm_fedora(
     worker_node1,
     unprivileged_client,
     namespace,
 ):
     with VirtualMachineForTests(
-        name=VM_CIRROS,
+        name=VM_FEDORA,
         namespace=namespace.name,
         client=unprivileged_client,
-        os_flavor=OS_FLAVOR_CIRROS,
+        os_flavor=OS_FLAVOR_FEDORA,
         memory_requests=Images.Cirros.DEFAULT_MEMORY_SIZE,
```

Comment:
If we switch to Fedora, the memory_request must also match that - e.g. use Images.Fedora.DEFAULT_MEMORY_SIZE instead.